glide.yaml

```diff
@@ -10,24 +10,38 @@ import:
   version: v1.13.1
   subpackages:
   - pkg/mount
+- package: github.com/docker/distribution
+  version: edc3ab29cdff8694dd6feb85cfeb4b5f1b38ed9c
+  subpackages:
+  - digestset
+  - reference
+- package: github.com/gogo/protobuf
+  version: c0656edd0d9eab7c66d1eb0c568f9039345796f7
+  subpackages:
+  - proto
+  - sortkeys
 - package: github.com/golang/glog
 - package: github.com/guelfey/go.dbus
+- package: github.com/juju/ratelimit
+  version: 5b9ff866471762aa2ab2dced63c9fb6f53921342
 - package: github.com/mitchellh/mapstructure
   version: db1efb5
 - package: gopkg.in/gcfg.v1
   version: v1.2.1
 - package: k8s.io/apiserver
-  version: kubernetes-1.8.2
+  version: 9707c008294d447053c0e2737c256ee23f99e5b4
 - package: k8s.io/apiextensions-apiserver
-  version: kubernetes-1.8.2
+  version: e509bb64fe1116e12a32273a2032426aa1a5fd26
 - package: k8s.io/apimachinery
-  version: kubernetes-1.8.2
+  version: 019ae5ada31de202164b118aee88ee2d14075c31
 - package: k8s.io/client-go
-  version: kubernetes-1.8.2
+  version: 35ccd4336052e7d73018b1382413534936f34eee
 - package: k8s.io/api
-  version: kubernetes-1.8.2
+  version: 4df58c811fe2e65feb879227b2b245e4dc26e7ad
```

Comment:
use a release tag instead? otherwise it's hard to tell whether the repos are in sync